### PR TITLE
Propose templating system #2

### PR DESCRIPTION
**Pull request by vihanb**
This branches off #14 and simplifies the templating system. Making a PR draft for everyone to look over and generate a preview

The structure is a:

 - [data](https://github.com/riceteacatpanda/homepage/tree/simplify-templating/src/_data) directory where all the data is stored and modified in one place/
 - [includes](https://github.com/riceteacatpanda/homepage/tree/simplify-templating/src/_includes) directory with the webpage templates
 - [static](https://github.com/riceteacatpanda/homepage/tree/simplify-templating/src/static) directory with all the static files to be copied to root

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,153 @@
 [![Netlify Status](https://api.netlify.com/api/v1/badges/9ffa72b9-e889-479d-95ca-4972dbfb5281/deploy-status)](https://app.netlify.com/sites/rtcp/deploys)
```
Comment (codemicro) [inline README.md]: Will need updating to provide a quick overview of the structure and layout of templates.

File: src/static/media-logos/instagram.svg [hunk 2]
```diff
@@ -1,67 +1,67 @@
-<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (codemicro) [inline src/static/media-logos/instagram.svg]: Not sure about having the Instagram logo in colour when all the other logos are in white.

**Comment by codemicro** [comment]
The navbar on mobile needs fixing - perhaps by replacing the items in the list with some sort of hamburger dropdown menu? 
image

**Comment by coc0a** [comment]
The Twitter icon also appears to be missing in that screenshot of the homepage.

**Comment by codemicro** [comment]
That's not because of the website, that's because of uBlock Origin.

**Pull request merged**